demo/JD-recommendation/load_data.sql

```diff
@@ -0,0 +1,8 @@
+USE JD_db;
+SET @@execute_mode='offline';
```

Comment:
Add SET @@job_timeout=600000;
The default timeout for every job is 20000ms(20 seconds), may meet with timeout error